### ISSUE TITLE
Concerns about the loss function

### ISSUE THREAD

**Issue by HaoKun-Li** [liuzechun/Nonuniform-to-Uniform-Quantization#4]
Hello, thanks for your excellent work and code! 

In the paper, the authors claim that they use the same knowledge distillation scheme as LSQ to train the quantized models. I show the screenshot as follows:
![image](https://user-images.githubusercontent.com/34161960/190353524-387edde3-4b77-416c-9a07-a38ac9897db9.png)

However, in the paper of LSQ, LSQ uses both the distillation loss function of Hinton et al. (2015) with temperature of 1 and equal weight given to the standard loss during training. I show the screenshot as follows:
![image](https://user-images.githubusercontent.com/34161960/190354260-17dc336a-b166-420e-b744-02d669474d3e.png)

When I read the code in this github. I notice that you have defined both the KD_loss and CrossEntropyLabelSmooth loss, but you use only distillation loss to train the quantized models. Is this a mistake, or a trick to improve the accuracy?

**Comment by szq0214** [comment]
Hi @HaoKun-Li Thanks for your interest in this work, the KD loss here is following [MEAL V2](https://github.com/szq0214/MEAL-V2) without the standard loss term, rather than LSQ scheme. It can provide better results.
